### PR TITLE
New package: ksuperkey-0.4

### PR DESCRIPTION
**Pull request by caioalonso**

### CODE REVIEW

File: srcpkgs/ksuperkey/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+pkgname=ksuperkey
+version=0.4
+revision=1
+build_style=gnu-makefile
+make_use_env=compliant
+hostmakedepends="pkg-config"
+makedepends="libX11-devel libXtst-devel"
+short_desc="Allows you to open the application launcher in KDE using the Super key"
+maintainer="Caio Alonso <caio@caioalonso.com>"
+license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/ksuperkey/template]: please use the spdx identifier

File: srcpkgs/ksuperkey/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+homepage="https://github.com/hanschen/ksuperkey"
+distfiles="https://github.com/hanschen/ksuperkey/archive/v${version}.tar.gz"
+checksum="0d202c0c7bc9a6eab5ce4f4049545df5ed277af9448361d94a5b725b864fcbbe"
```
Comment (maxice8) [inline srcpkgs/ksuperkey/template]: checksum doesn't need to be quoted

File: srcpkgs/ksuperkey/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'ksuperkey'
+pkgname=ksuperkey
+version=0.4
+revision=1
+build_style=gnu-makefile
+make_use_env=compliant
```
Comment (cr6git) [inline srcpkgs/ksuperkey/template]: Interesting.
Out of curiosity, what `make_use_env=compliant` is supposed to do?

**Comment by maxice8** [comment]
Isn't there a manpage available ? would be nice to have.

File: srcpkgs/ksuperkey/template (hunk 3 above)
Comment (cr6git) [reply srcpkgs/ksuperkey/template]: I don't see `make_use_env=compliant` in the Manual.

**Comment by maxice8** [comment]
On Mon, Oct 08, 2018 at 08:32:55PM -0700, cr6git wrote:
> cr6git commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,14 @@
> +# Template file for 'ksuperkey'
> +pkgname=ksuperkey
> +version=0.4
> +revision=1
> +build_style=gnu-makefile
> +make_use_env=compliant
> 
> I don't see `make_use_env=compliant` in the Manual.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/3447#discussion_r223551979

most common uses for make_use_env is compliant and yes

rg --count -- 'make_use_env=yes' srcpkgs/ | awk 'BEGIN{FS=":"}{x+=$2}END{print x}'
34

rg --count -- 'make_use_env=compliant' srcpkgs/ | awk 'BEGIN{FS=":"}{x+=$2}END{print x}'
5

rg --count -- 'make_use_env=1' srcpkgs/ | awk 'BEGIN{FS=":"}{x+=$2}END{print x}'
7

**Comment by caioalonso** [comment]
> Isn't there a manpage available ? would be nice to have.
Sorry, no manpage available at https://github.com/hanschen/ksuperkey. I could create one, but it wouldn't be available in the current release.

**Pull request merged**